site/content/docs/5.1/components/navs-tabs.md

```diff
@@ -175,49 +175,51 @@ Nav tabs light only differ visually, with a full width bottom border and a diffe
 Nav tabs light is nested in a tab for adding a level of depth in information organization.
 
 {{< example >}}
-<div role="tablist" aria-owns="nav-tab1 nav-tab2 nav-tab3 nav-tab4">
-  <ul class="nav nav-tabs" id="nav-tab-with-nested-tabs">
-    <li class="nav-item">
-      <a class="nav-link active" aria-current="page" id="nav-tab1" href="#tab1-content" data-bs-toggle="tab" data-bs-target="#tab1-content" role="tab" aria-controls="tab1-content" aria-selected="true">Tab 1</a>
-    </li>
-    <li class="nav-item">
-      <a class="nav-link" id="nav-tab2" data-bs-toggle="tab" href="#tab2-content" data-bs-target="#tab2-content" role="tab" aria-controls="tab2-content" aria-selected="false">Tab 2</a>
-    </li>
-    <li class="nav-item">
-      <a class="nav-link" id="nav-tab3" data-bs-toggle="tab" href="#tab3-content" data-bs-target="#tab3-content" role="tab" aria-controls="tab3-content" aria-selected="false">Tab 3</a>
-    </li>
-    <li class="nav-item">
-      <a class="nav-link disabled" id="nav-tab4" role="tab" aria-selected="false">Tab 4</a>
-    </li>
-  </ul>
-</div>
-
-<div class="tab-content" id="nav-tabs-content">
-  <div class="tab-pane-with-nested-tab fade show active" id="tab1-content" role="tablist" aria-labelledby="nav-tab1">
-    <ul class="nav nav-tabs nav-tabs-light mt-0">
+<div class="nav-tabs border-bottom-0">
```

Comment:
Regarding the use of CSS vars, it seemed more logical to have a surrounding `<div>` with `.nav-tabs`.

Reply:
Yup, I don't know if there's a better way to handle it, need to discuss about it, because as-is, we use `.nav-tabs` twice in a row

Reply:
Reverting it since Bootstrap doesn't do it like this